src/dashboards/Timeline/TimelineDashboard.tsx

```diff
@@ -1,494 +1,21 @@
-import React, { Fragment, useMemo, useEffect, useState, useRef } from 'react'
-import { useParams } from 'react-router-dom'
-import { useLongPress } from 'use-long-press'
-import { Heading2 } from '@entur/typography'
-import { LegBone } from '@entur/travel'
-import { colors } from '@entur/tokens'
-import { WalkingIcon } from '@entur/icons'
-import { BREAKPOINTS } from '../../constants'
-import { LineData, IconColorType } from '../../types'
-import { useStopPlacesWithDepartures, useWalkInfo } from '../../logic'
+import React from 'react'
```

Comment:
Så mye redusert i størrelse at github ikke viste diff by default fordi den var for stor, goooood stuff